draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -582,8 +602,9 @@ To this end, Verifiers MUST follow the following steps (or equivalent):
        2. Determine the public key for the Holder from the SD-JWT.
        3. Ensure that a signing algorithm was used that was deemed secure for the application. Refer to [@RFC8725], Sections 3.1 and 3.2 for details. The `none` algorithm MUST NOT be accepted.
        4. Validate the signature over the Holder Binding JWT.
-       5. Check that the Holder Binding JWT is valid using `nbf`, `iat`, and `exp` claims, if provided in the Holder Binding JWT.
-       6. Determine that the Holder Binding JWT is bound to the current transaction and was created for this Verifier (replay protection). This is usually achieved by a `nonce` and `aud` field within the Holder Binding JWT.
+       5. Check that the `typ` of the Holder Binding JWT is `hb+jwt`.
```

Comment:
Why is `hb+jwt` needed for this? We would also need to register a media type -> application/hb+jwt for this.

Reply:
yes, that's the plan, to register

Reply:
I added the IANA registration section in this PR :)